web/client/epics/map.js

```diff
@@ -259,6 +259,7 @@ const compareMapChanges = (action$, { getState = () => {} }) =>
                 groupsSelector(state),
                 backgroundListSelector(state),
                 textSearchConfigSelector(state),
+                bookmarkSearchConfigSelector(state),
```

Comment:
will this change need a dedicated unit test?

Reply:
Same as above